vm/actor_interface/src/builtin/miner/mod.rs

```diff
@@ -60,8 +60,7 @@ impl State {
     pub fn info<BS: BlockStore>(&self, store: &BS) -> anyhow::Result<MinerInfo> {
         match self {
             State::V8(st) => {
-                let fvm_store = ipld_blockstore::FvmRefStore::new(store);
-                let info = st.get_info(&fvm_store)?;
+                let info = st.get_info(&store)?;
```

Comment:
`store` is already a reference, right?
```suggestion
                let info = st.get_info(store)?;
```